### ISSUE TITLE
Specifying download directory doesn't work on Ubuntu 18.04

### ISSUE THREAD

**Issue by cobalt2727** [Bijman/srb2dl#6]
Getting the following error when I try installing anything:
```ruby
Downloading chosen files. Please wait..
curl: option --output-dir: is unknown
curl: try 'curl --help' or 'curl --manual' for more information

Downloading is successful.
```

There's a pretty simple explanation for this: https://daniel.haxx.se/blog/2020/09/10/store-the-curl-output-over-there/
`tldr: --output-dir [directory] comes in curl 7.73.0`

Interesting.
```bash
ubuntu@ubuntu-switch-4-cores-4-gb:~$ curl --version
curl 7.58.0 (aarch64-unknown-linux-gnu) libcurl/7.58.0 OpenSSL/1.1.1 zlib/1.2.11 libidn2/2.0.4 libpsl/0.19.1 (+libidn2/2.0.4) nghttp2/1.30.0 librtmp/2.3
Release-Date: 2018-01-24
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtmp rtsp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz TLS-SRP HTTP2 UnixSockets HTTPS-proxy PSL 
```

No wonder - Ubuntu 18.04 is too old for the parameter you use to determine download location.
You've got a few options here:
1. Make the program explicitly require that the OS has the ability to use `curl --output-dir` - which would be totally fair, my VM is only on something this old for [testing purposes](https://github.com/cobalt2727/L4T-Megascript) anyways
2. Find some other alternative to specify download locations

Regardless, you should set up some sort of error handling if the download fails, e.g. `[download command goes here] || error message goes here` so that it's not just claiming the download succeeds when it hasn't.

**Comment by Bijman** [comment]
Thanks for the feedback. Alternative method of downloading files to directory, to be implemented to the script, would be using only "-o" option, which would have full path with filename, for example `curl -LC - [fileURL] -o [path to directory]/[filename]`.

**Comment by Bijman** [comment]
Problem solved with commit 801aaca.
